src/components/hub/elements/MessageInput.js

```diff
@@ -42,6 +43,36 @@ export default function MessageInput({
     [onSubmitFunc]
   );
 
+  const uploadInputText = useCallback(() => {
+    if (inputTextRef.current.value) {
+      localStorage.setItem("mtext", inputTextRef.current.value);
+      inputTextRef.current.value = "";
+      inputTextRef.current.style.height = `55px`;
+    }
+  }, [inputTextRef]);
```

Comment:
Using ref as a dependency wont' trigger useCallback/useEffect/useMemo  because mutating the reference won't trigger a render.

In this particular case we can remove useCallback, cause the function is pretty simple